generator/generator.go

```diff
@@ -430,6 +434,10 @@ func main() {
 		ListBuilderKeyThreshold: *listBuilderKeyThreshold,
 		GenerateWildcardPaths:   *generateWildcardPaths,
 		SimplifyWildcardPaths:   *simplifyWildcardPaths,
+		TrimOCPackage:           *trimOCPackage,
+		FakeRootPackageName:     *fakeRootPackageName,
+		SplitByModule:           *splitByModule,
+		BaseImportPath:          *baseImportPath,
 		InputModulesOnly:        *inputModulesOnly,
```

Comment:
Add a flag validation that `splitByModule=true` only if `inputModulesOnly=true` as well.